scripts/jenkins/jobs-ibs/cloud-mediacheck.yaml

```diff
@@ -24,6 +24,7 @@
             - SUSE:SLE-11-SP3:Update:Cloud5:Test
             - SUSE:SLE-12-SP1:Update:Products:Cloud6/SLE_12_SP1
             - SUSE:SLE-12-SP2:Update:Products:Cloud7/SLE_12_SP1
```

Comment:
That's fine because we use a hack in the build service to build SP2 packages against SP1. So it looks like SP2 but it is SP1.